cif2cell/cif2cell.xml

```diff
@@ -38,6 +125,9 @@
     </tests>
```

Comment:
We should have tests that use the new parameters, e.g. a file that has used `--no-reduce`, one that used `--vca` and one that has used `--cartesian`